nf_core/pipeline-template/.github/workflows/awsfulltest.yml

```diff
@@ -19,19 +19,29 @@ jobs:
     if: github.repository == '{{ name }}' && github.event.review.state == 'approved' && github.event.pull_request.base.ref == 'master' || github.event_name == 'workflow_dispatch'
     runs-on: ubuntu-latest
     steps:
-      - uses: octokit/request-action@v2.x
+      - name: Get PR reviews
+        uses: octokit/request-action@v2.x
         if: github.event_name != 'workflow_dispatch'
         id: check_approvals
         with:
           route: GET /repos/{%- raw -%}${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews?per_page=100
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - id: test_variables
+
+      - name: Check for approvals
```

Comment:
I think it won't be run if the previous one failed? We probably need to add a `continue-on-error: true`